desc/plotting.py

```diff
@@ -3061,7 +3336,12 @@ def plot_field_lines_real_space(
     ax.set_zlim3d([z_middle - plot_radius, z_middle + plot_radius])
 
     if return_B_interp:
-        return fig, ax, field_line_coords, B_interp
+        return (
+            fig,
+            ax,
+            field_line_coords,
+            B_interp,
+        )
     else:
         return fig, ax, field_line_coords
```

Comment:
might make sense to not return field line coords here, to be consistent with the others